pkgs/top-level/all-packages.nix

```diff
@@ -7419,6 +7419,8 @@ with pkgs;
 
   memtester = callPackage ../tools/system/memtester { };
 
+  mergedb = python3Packages.callPackage ../development/python-modules/mergedb { };
```

Comment:
`mergedb` is a Python module and should be added to `python-packages.nix` instead of `all-packages.nix`.